### PR TITLE
Added intel project oneCCL

### PR DESCRIPTION
**Pull request by sudhirverma**
Issue: #376

### CODE REVIEW

**Review by asa1997 — CHANGES_REQUESTED** [review]
@sudhirverma version detail file name is incorrect

File: bes_theme/assets/data/version_details/376-oneCCL-Versiondetails.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "version": "2021.7.1",
+        "release_date": "",
```
Comment (asa1997) [inline bes_theme/assets/data/version_details/376-oneCCL-Versiondetails.json:4]: @sudhirverma Please provide the release date

**Review by asa1997 — CHANGES_REQUESTED** [review]
@sudhirverma Please address the comments

File: bes_theme/assets/data/version_details/376-oneCCL-Versiondetails.json (hunk 1 above)
Comment (sudhirverma) [reply bes_theme/assets/data/version_details/376-oneCCL-Versiondetails.json]: done

**Review by asa1997 — APPROVED** [review]
@sudhirverma LGTM.

**Pull request merged**